lib/optimizely.rb

```diff
@@ -282,6 +282,28 @@ def is_feature_enabled(feature_flag_key, user_id, attributes = nil)
       false
     end
 
+    def get_enabled_features(user_id, attributes = nil)
+      # Returns the list of features that are enabled for the user.
+      # Args:
+      #   user_id: ID for user.
+      #   attributes: Dict representing user attributes.
+      # Returns:
+      #   A list of the keys of the features that are enabled for the user.
+      #
+      enabled_features = []
+
+      unless @is_valid
+        logger = SimpleLogger.new
+        logger.log(Logger::ERROR, InvalidDatafileError.new('get_enabled_features').message)
+        return enabled_features
+      end
+
+      @config.feature_flags.each do |feature|
+        enabled_features.push(feature['key']) if is_feature_enabled(feature['key'], user_id, attributes)
```

Comment:
Please only push feature key when is_feature_enabled returns strictly 'true'. 
FYI - In Ruby only false and nil are falsey. Everything else is truthy (yes, even 0 is truthy).